### ISSUE TITLE
Href attributes are missing from the Navbar links

### ISSUE THREAD

**Issue by MarikaKononen** [devgeniem/bulmally#1]
From the accessibility point of view all Navbar links should have href attributes. Those are now missing and accessibility cannot be tested.

**Comment by MarikaKononen** [comment]
Fixed in branch issue-1.
